pkg/local_object_storage/writecache/delete.go

```diff
@@ -20,6 +20,7 @@ func (c *cache) Delete(addr *objectSDK.Address) error {
 			copy(c.mem[i:], c.mem[i+1:])
 			c.mem = c.mem[:len(c.mem)-1]
 			c.mtx.Unlock()
+			c.curMemSize -= uint64(len(c.mem[i].data))
```

Comment:
This should be done with mutex taken (before `c.mtx.Unlock`).

Reply:
Done.